examples/clock.js

```diff
@@ -50,28 +50,29 @@ function numbers(x, y, radius) {
 
 // draw one of the clock hands
 function hand (theta, w, length, col) {
-   var stepSize = 5;
-   var widthDelta = w / (length / stepSize);
+   const stepSize = 5;
+   const widthDelta = w / (length / stepSize);
```

Comment:
Those parens () aren't needed:
`const widthDelta = w / length / stepSize;`

Reply:
Are you sure?
```js
1 / 2 / 3 //0.16666666666666666
1 / (2 / 3) //1.5
```

Reply:
Dunno where my brain was that somehow I viewed it as if those parens were somehow at the beginning of the expression. Just ignore this 1! :face_with_spiral_eyes:

Reply:
Ok lol, that sometimes happens to me